examples/pos/capture/ffi_blocks.effekt

```diff
@@ -1,6 +1,6 @@
 interface Foo {}
-extern io def something {f: Foo}: Int = "${box f}"
-extern io def createFoo(): Foo at {global} = "42"
+extern io def something {f: Foo}: Int = simpleAtom "${box f}"
+extern io def createFoo(): Foo at {global} = simpleAtom "42"
```

Comment:
This is just a bad test (already has been), not your fault

Reply:
I tried to fix it up, and it found a bug: We cannot actually call any operations on `f` since the capture is not allowed by our current implementation of Typer here.
Just allowing this in typer, however, leads to crashes in the backends (more precisely: undefined variables). I think this might be related to not passing the captures to the extern 🤔.